### PR TITLE
Added setup dependencies from muse_deps (macos)

### PR DESCRIPTION
**Pull request by igorkorsukov**
Includes:
* Getting dependencies from muse_dep for MacOS
* Build MacOS in CI

not working yet - dmg packaging, will be done next PR

### CODE REVIEW

File: .github/workflows/ci_macos_mu4.yml
Comment (Eism) [inline .github/workflows/ci_macos_mu4.yml:1]: should be au4

File: au4/buildscripts/packaging/MacOS/MacOSXBundleInfo.plist.in
Comment (Eism) [inline au4/buildscripts/packaging/MacOS/MacOSXBundleInfo.plist.in:1]: can we reuse [/cmake-proxies/cmake-modules/MacOSXBundleInfo.plist.in](https://github.com/Eism/audacity/blob/master/cmake-proxies/cmake-modules/MacOSXBundleInfo.plist.in) ?

File: .github/workflows/ci_macos_mu4.yml
Comment (igorkorsukov) [reply .github/workflows/ci_macos_mu4.yml:1]: fixed

File: au4/buildscripts/packaging/MacOS/MacOSXBundleInfo.plist.in
Comment (igorkorsukov) [reply au4/buildscripts/packaging/MacOS/MacOSXBundleInfo.plist.in:1]: copied this plist

**Review by Eism — APPROVED** [review]

**Pull request merged**